apache2/apache2_util.c

```diff
@@ -344,6 +344,14 @@ static void send_waf_log(const char* timestamp, struct waf_lock* lock, apr_file_
     get_short_filename(waf_filename);
     get_ruleset_type_version(waf_ruleset_info, waf_ruleset_type, waf_ruleset_version); 
 
+    // Format UTC time
+    time_t rawtime;
+    struct tm * timeinfo;
```

Comment:
`... = NULL;`

Reply:
`struct tm * timeinfo = NULL`? Done

Reply:
actually, changing to not be a ptr now that I'm using gmtime_r.